.github/workflows/deploy.yml

```
@@ -0,0 +1,50 @@
+                SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+                SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+                SLUG: dokan-lite
+            - name: WordPress.org plugin asset/readme update
+              uses: 10up/action-wordpress-plugin-asset-update@stable
+              env:
+                SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+                SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+                IGNORE_OTHER_FILES: true
+                SLUG: dokan-lite
+            - name: Create GitHub release
+              uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)